### ISSUE TITLE
ODF issue on attenuation unit

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1572]
See below. I'll fix this after noon.

```
8: In ODFNames2oceNames(ODFnames, ODFunits, PARAMETER_HEADER = NULL,  :
  unable to interpret ODFunits[9]='/m', for item named 'ATTU'
Execution halted
```

**Comment by richardsc** [comment]
Seems odd that the default is to halt execution, rather than just spit a warning.

**Comment by dankelley** [comment]
This error is encountered because we use `parse()` on the string, which cannot be done, e.g.
```R
> parse(text="1/m")
expression(1/m)
```
works but
```R
> parse(text="/m")
Error in parse(text = "/m") : <text>:1:1: unexpected '/'
1: /
```
(as in the ODF file) fails.

I'll fix this by

1. doing `gsub("^/", "1/", unitName)`
2. making things more robust, by trying to use `parse` on any unrecognized unit, but wrapping that in a `try` so we can simply copy the string, if we need to.

**Comment by dankelley** [comment]
Fixed in "develop" branch, commit 9201141c55aa76f2e2abe275bcf92c75e3b05a03
